### PR TITLE
Fix about dialog on Qt 5

### PR DESCRIPTION
**Pull request by estan**
QMessageBox is in QtWidgets, not QtGui.

### CODE REVIEW

File: examples/user_interfaces/embedding_in_qt5.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def closeEvent(self, ce):
         self.fileQuit()
 
     def about(self):
-        QtGui.QMessageBox.about(self, "About",
+        QtWidgets.QMessageBox.about(self, "About",
                                 """embedding_in_qt5.py example
```
Comment (QuLogic) [inline examples/user_interfaces/embedding_in_qt5.py]: This line needs to be re-aligned with the opening parentheses above.

Comment (estan) [inline examples/user_interfaces/embedding_in_qt5.py]: Ah, sorry about that. Commit updated. I also removed the now unused import of `QtGui`.

**Comment by tacaswell** [comment]
:+1: Thanks

**Comment by tacaswell** [comment]
backported to v1.5.1-doc as 20db6a0

**Pull request merged**